src/smart-components/MessageSearch/index.scss

```diff
@@ -95,6 +95,6 @@
     box-sizing: border-box;
     width: 100%;
     height: 100%;
-    overflow-y: scroll;
+    overflow-y: hidden;
```

Comment:
What happened here?

Reply:
The outer component is set by scroll, so the internal component wasn't applied scroll